pinaxcon/templates/static_pages/homepage.html

```diff
@@ -24,20 +24,21 @@
         <h3>A Python conference north of the Golden Gate</h3>
       </div>
 
-      <div class="bottom-align">
         <h3>
           The Mystic Theatre</br>
           Petaluma, California
         </h3>
         <h4>
-          <strong>2017 conference:</strong> 2 &amp; 3 December <br/>
-          Talk submissions close 29 September
+          <strong>2017 conference:</strong> December 2 &amp; 3 <br/>
+          Talk submissions close September 29
         </h4>
 
-        <div style="height:0.5em;"></div>
+      </div>
+
+      <div class="container homepage-block-footer">
 
         <div class="btn-group">
-          <a class="btn btn-lg btn-primary" href="/dashboard">Submit a proposal</a>
+          <a class="btn btn-lg btn-primary btn-shadow" href="/dashboard">Submit a proposal</a>
```

Comment:
need to add title tag. (also, oops, may need to add title tag to button on call for proposals page as well)